mac-setup.sh

```diff
@@ -205,9 +205,8 @@ install_homebrew() {
     brew install apple-gcc42
 
     # We need a particular version of phantomjs to run the end-to-end tests.
+    # This command will let us access those.
     brew tap homebrew/versions
-    brew uninstall --force phantomjs
-    brew install phantomjs198
```

Comment:
oh huh! yeah, we should have moved this into the install_phantomjs function. good catch :+1: